bots/psuteam7bot/castle.js

```diff
@@ -167,7 +175,8 @@ castle.findPosition = (self) => {
                 y: 0, 
                 buildCounter: buildCounter, 
                 signalBuilding: false,
-                mirrorCastleDestroyed: false
+                mirrorCastleDestroyed: false,
+                silentCount: 0
```

Comment:
Guess `silentCount` and receiving `castle_talk` acts as timeout check? Nice addition.